src/Admin/OptionsMenu.php

```diff
@@ -325,4 +324,26 @@ public function render_screen() {
 		</div>
 		<?php
 	}
+
+	/**
+	 * Adds REST paths to preload.
+	 *
+	 * @param array  $paths Array of paths that will be preloaded on the backend.
+	 * @param string $screen The current screen ID.
+	 * @return array Filtered paths.
+	 */
+	public function filter_preload_rest_paths( $paths, $screen ) {
```

Comment:
There's a lot of non-DRYness with this class and OnboardingWizardSubmenuPage.php. I'm sure they could be refactored in some way, but I'm not sure of the best approach yet, or if now's the time to do it.